lib/westfield_datetime_helper/validators/zulu_time.rb

```diff
@@ -0,0 +1,35 @@
+# Validates that a supplied value is in fact
+# a DateTime, and is in timezone UTC+0
+require 'time'
+require 'active_model'
+
+module WestfieldLabs
+  module DatetimeHelper
+    module Validators
```

Comment:
Not a bad idea.  I'll defer all such decisions however until the core is working.